src/Discord.Net.Rest/Entities/AuditLogs/DataTypes/ScheduledEventUpdateAuditLogData.cs

```diff
@@ -51,16 +52,15 @@ internal static ScheduledEventUpdateAuditLogData Create(BaseDiscordClient discor
                 status?.OldValue.ToObject<GuildScheduledEventStatus>(discord.ApiClient.Serializer),
                 entityType?.OldValue.ToObject<GuildScheduledEventType>(discord.ApiClient.Serializer),
                 entityId?.OldValue.ToObject<ulong?>(discord.ApiClient.Serializer),
-                entityMetadata?.OldValue.ToObject<GuildScheduledEventEntityMetadata>(discord.ApiClient.Serializer)
-                    ?.Location.GetValueOrDefault(),
+                location == null ? null : location.OldValue?.ToObject<string>(discord.ApiClient.Serializer) ?? Optional<string>.Unspecified,
```

Comment:
Same here

Reply:
you literally cannot convert this in a null coalescing expression because of bracket and calling rules